### DISCUSSION TITLE
Is it possible to add a comment field from a torrent file to ruTorrent if the torrent was added via rtorrent Watch Directory Schedules?

### DISCUSSION THREAD

**Discussion by omgiafs** [Novik/ruTorrent#2527] [Q&A]
Hello! 

Most trackers add a link to the tracker topic where torrent file was downloaded from in the comment field in the torrent file. 

It is very convenient when in the ruTorrent interface in the "Details" menu there is a comment with a link to the tracker topic. 
In this case, when torrent updated on the tracker, you can go to the topic on tracker's forum and download the updated torrent with almost one click. 

If the torrent file is added to rtorrent manually through the ruTorrent interface, then there is no problem:

![add_torrent_with_ruTorrent](https://github.com/Novik/ruTorrent/assets/7763953/dff094c1-864a-46a0-8f7b-22f745dbe0a7)

However, if you add a torrent file through the rtorrent Watch Directory Schedules mechanism, then the comment field in the ruTorrent interface is empty:

![add_torrent_with_rtorrent_watch_folders](https://github.com/Novik/ruTorrent/assets/7763953/83fb3f39-a378-4909-ab32-3c2b04139b55)

As far as I understand, rtorrent ignores this field (why?), and ruTorrent in this case just takes all the data which are in rtorrent for this torrent. 

In this regard, I have two questions: 

1. How to automatically add a comment to the ruTorrent Torrent() object, if the distribution was added to rtorrent via Watch Directory Schedules? 
2. Is it possible to automatically download the updated seed torrent file using ruTorrent and add it to rtorrent? In this case, an empty comment field would not be a problem. 

I strongly suspect that all these processes have been automated already. But there is almost no structured information about this anywhere. I would be grateful if you share links to such manuals.

**Comment by stickz** [comment]
You can use the _**Autotools**_ plugin on **ruTorrent**. There is an `Auto Watch` feature.

English: https://github.com/Novik/ruTorrent/wiki/PluginAutotools
Russian: https://github.com/Novik/ruTorrent/wiki/PluginAutotools.ru
